supplier_product/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+from product.models import Product
+from supplier.models import Supplier
+
+
+class SupplierProduct(models.Model):
+    supplier_product_id = models.AutoField(primary_key=True)
+    qr_code = models.ForeignKey(Product, on_delete=models.CASCADE)
+    user_name = models.ForeignKey(Supplier, on_delete=models.CASCADE)
+    price = models.IntegerField(validators=[MinValueValidator(1)])
+    quantity = models.IntegerField()
```

Comment:
We need here a MinValueValidator set to zero, to verify we don't get negative values.  
```suggestion
    quantity = models.IntegerField(validators=[MinValueValidator(0)])
```